templates/dhcpd.host.erb

```diff
@@ -21,6 +21,11 @@ host <%= @host %> {
   option <%= option %> <%= @options[option] %>;
 <% end -%>
 <% end -%>
+<% if not @parameters.empty? -%>
```

Comment:
this this is always an hash, you could just omit the if condition and directly loop across the variable